pkgs/development/python-modules/stravaweblib/default.nix

```diff
@@ -0,0 +1,44 @@
+{
+  lib,
+  arrow,
+  pythonOlder,
+  buildPythonPackage,
+  fetchFromGitHub,
+  stravalib,
+  beautifulsoup4,
+  setuptools,
+  setuptools-scm,
+}:
+
+buildPythonPackage rec {
+  pname = "stravaweblib";
+  version = "0.0.8";
+
+  disabled = pythonOlder "3.4";
```

Comment:
I don't think anyone will ever try to use Python 3.4 in the near future with Nixpkgs. I'd remove this, to simplify things.

Reply:
Addressed in 57b4a24f4d4d